Makefile

```diff
@@ -162,7 +164,7 @@ receptorctl-test-venv/bin/pytest:
 	receptorctl-test-venv/bin/pip install -r receptorctl/test-requirements.txt
 
 receptorctl-tests: receptor receptorctl-test-venv/bin/pytest
-	cd receptorctl && ../receptorctl-test-venv/bin/pytest tests/tests.py
+	cd receptorctl && ../receptorctl-test-venv/bin/pytest --junitxml=../receptorctl-test-junit.xml tests/tests.py
 
 clean:
 	@rm -fv receptor receptor.exe receptor.app net $(SPECFILES)
```

Comment:
Make clean should remove test-output.log, test-junit.xml and receptorctl-test-junit.xml.  These should also be added to .gitignore.